api/build.gradle

```diff
@@ -25,13 +80,19 @@ dependencies {
   compile('org.springframework.boot:spring-boot-starter-web:+') {
     exclude module: 'spring-boot-starter-tomcat'
   }
+  compile 'org.springframework.security:spring-security-web:+'
```

Comment:
Done. (I switched to + instead of 2.2.2... I was experimenting with different versions earlier, but the latest version works fine.)